app/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getAllTopics } = require('./app.controllers')
+const { handleNotFoundError } = require('./errors')
+
+const app = express();
+app.use(express.json());
```

Comment:
We do not need app.use(express.json( )) - we are not dealing with POST/PATCH requests yet.